proprietary/tokens/src/components/amsterdam/mega-menu.tokens.json

```diff
@@ -0,0 +1,15 @@
+{
+  "amsterdam": {
+    "mega-menu": {
+      "padding-block": {
+        "value": "clamp(2.5rem, calc(2.2917rem + 1.0417vw), 3rem) clamp(3.5rem, calc(3.2917rem + 1.0417vw), 4rem)"
+      },
+      "list-wrapper": {
+        "column-gap": { "value": "{amsterdam.grid.spacious.gap}" },
+        "column-width": { "value": "20rem" },
```

Comment:
This value introduces a narrow window between 1088px and 1174px, where we have twelve columns, but the first list is still in one column. It’s a bit magic number-y, as I saw Aram comment somewhere.

How about ‘4 * grid column width + 3 * grid column gap’? That’s precisely what we want to express, right? Splitting or merging of lists should happen at our grid breakpoints.

Reply:
Also, the issue of the narrow window appeared in the other PR (#1020) when MegaMenu was placed in the `<Screen>` component and the Header. So perhaps we should adress this in the other PR? To tweak this value?

Reply:
As discussed, this approach will stay and value has changed to 22rem.